src/app/app.component.scss

```diff
@@ -0,0 +1,74 @@
+#buttonDiv{
+  width:100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: center;
+  background-color: #2a2a2a;
+  border-top: black 1px solid;
+  border-bottom: black 1px solid;
+}
+#buttonDiv button{
+  min-height:40px;
+  max-width:250px;
+  color:#fff;
+  background-color:  #3b3b3b ;
```

Comment:
Not that it really matters, but one space after the colon and before the value makes it much easier to read.

Same for variables in JS code 😄

Reply:
Ill try start doing it this way, makes more sense especially since ive not been consistent with value & variable spacing. ill try sort out a formality :p

Reply:
Do you use a linter?

Reply:
`crtl + alt +L` in intellij should sort this kinda stuff out for you 👍